config.json

```diff
@@ -671,26 +890,34 @@
       ]
     },
     {
+      "uuid": "790d60b6-a64e-482f-8c2b-3e2442ac0b4e",
       "slug": "change",
+      "core": false,
+      "unlocked_by": null,
       "difficulty": 4,
       "topics": [
         "control-flow (if-else statements)",
         "control-flow (loops)"
       ]
     },
     {
+      "uuid": "a608af44-59df-425d-a652-163635bf740e",
       "slug": "secret-handshake",
+      "core": false,
+      "unlocked_by": null,
       "difficulty": 3,
       "topics": [
         "control-flow (if-else statements)",
         "arrays",
         "bitwise operations"
       ]
+    },
+    {
+      "uuid": "121732a1-e3a3-4791-9b83-ec8a61969d85",
+      "slug": "accumulate",
+      "deprecated": true
```

Comment:
Okay, I see now that this matches the description in https://github.com/exercism/meta/issues/16